kep/tbd-peaks-power-aware-scheduling/README.md

```diff
@@ -0,0 +1,326 @@
+    - Since energy savings decrease with the increased use of energy ineffecient nodes, PEAKS plugin favours pod placement on energy efficient nodes over pod placement on energy inefficient nodes.
+  - Below graph demonstrates energy saving while auto scaling pods with HPA controller ![PEAKS with HPA](./figs/PEAKS_with_HPA.png)
+    - The graph in the left depicts spreading of pods on nodes using default kube-scheduler.
+    - The graph in the middle depicts packing of pods on energy efficient nodes before pod placement on energy (relatively) inefficient cluster nodes using PEAKS scheduler plugin.
+    - The graph in the right depicts the savings in energy across the cluster nodes over time (i.e., difference in the aggregate cluster energy consumed under PEAKS plugin placement vs. default scheduler placement) as the pods scale up.
+    - There is ~10\% of energy savings observed at the end of 15 minutes of workload execution in this experiment.
+    - Note that the difference in the aggregate energy consumption of the cluster nodes reduces (highlighted in red circle) as PEAKS plugin places pods on energy inefficient nodes.
+- Kubectl scale command (while resizing of a deployment, replica set, replication controller, or stateful set):
+  - On a K8s cluster with heterogeneous node configurations (resulting in the energy efficiency of the nodes not being the same), scaling a pod of a deployment, replica set, replication controller, or stateful set with `kubectl scale` command packs the pods on nodes using PEAKS scheduler plugin vs. spreads the pods on nodes using default kube-scheduler.
+    - Since energy savings decrease with the increased use of energy ineffecient nodes, PEAKS plugin favours pod placement on energy efficient nodes over pod placement on energy inefficient nodes.
+  - The experiment below demonstrates energy savings while scaling pods with `kubectl scale` command.
+    - The pods of a deployment "cpu-stress-test" were initially placed on the energy efficient node "tantawi1". Exercising `kubectl scale` to increase the number of pods from 2 to 5 spreads the newly created pods across the nodes "tantawi1" and "tantawi2" irrespective of their energy effeciency with default kube-scheduler. ![Kubectl-scale_Default](./figs/Kubectl-scale_Default.png)
+    - The pods of a deployment "cpu-stress-test" were initially placed on the energy efficient node "tantawi1". Exercising `kubectl scale` to increase the number of pods from 2 to 5 packs the newly created pods on the energy efficient node "tantawi1" with PEAKS scheduler plugin. ![Kubectl-scale_PEAKS](./figs/Kubectl-scale_PEAKS.png)
+    - While `kubectl scale` is used to decrease the number of pods, scheduler (PEAKS plugin) doesn't have any role to play. Hence, pods running on on more energy efficient nodes may get evicted. One may use pod priority and preemption to evict pods from less energy efficient nodes as a workaround.
+- Migration of a Pod (via explicit eviction)
```

Comment:
Is this the same term of "de-scheduling"?

Reply:
Hi @Huang-Wei , it is not exactly the same as `de-scheduling`. However, `de-scheduling` is also relevant here. Hence, we have updated the description appropriately to include `de-scheduling`.